src/main/java/com/iota/iri/crypto/PearlDiver.java

```diff
@@ -22,22 +30,15 @@
     private volatile State state;
     private final Object syncObj = new Object();
 
-    public void cancel() {
-        synchronized (syncObj) {
-            state = State.CANCELLED;
-        }
-    }
-
-    private static void validateParameters(byte[] transactionTrits, int minWeightMagnitude) {
-        if (transactionTrits.length != TRANSACTION_LENGTH) {
-            throw new RuntimeException(
-                "Invalid transaction trits length: " + transactionTrits.length);
-        }
-        if (minWeightMagnitude < 0 || minWeightMagnitude > CURL_HASH_LENGTH) {
-            throw new RuntimeException("Invalid min weight magnitude: " + minWeightMagnitude);
-        }
-    }
-
+    /**
+     * Searches for a nonce such that the hash ends with {@code minWeightMagnitude} zeros.<br>
+     *
+     * @param transactionTrits trits of transaction
+     * @param minWeightMagnitude target weight for trailing zeros
+     * @param numberOfThreads number of worker threads to search for a nonce
+     * @return {@code true} if search completed successfully.
```

Comment:
I started to use `<tt>` for `true` and `false` becuase this is how I saw it in Oracle's javadocs.

Personally, I care less about this, as long as the resulting output looks good.